lib/amazonka/src/Amazonka/Auth/ConfigFile.hs

```diff
@@ -138,6 +139,11 @@ fromFilePath profile credentialsFile configFile env = liftIO $ do
                 fromAssumedRole roleArn "amazonka-assumed-role" sourceEnv
               AssumeRoleWithWebIdentity roleArn mRoleSessionName tokenFile ->
                 fromWebIdentity tokenFile roleArn mRoleSessionName env
+              SSO startUrl ssoRegion accountId roleName -> do
+                cache <-
+                  liftIO $
+                    configPathRelative =<< relativeCacheFile startUrl
```

Comment:
Open to ideas here. Passing the `startUrl` to `fromSSO` and having it call `configPathRelative` internally would make much more sense to me, but causes an import loop. This was the best I could come up with.

Reply:
I see by your second comment you mean `cache`. Done.